client/server/middleware/unsupported-browser.js

```diff
@@ -28,15 +56,7 @@ function allowPath( path ) {
 		? path.replace( new RegExp( `^/${ prefixedLocale }` ), '' )
 		: path;
 
-	const allowedPaths = [
-		'/browsehappy',
-		'/log-in',
-		'/start',
-		'/new',
-		'/themes',
-		'/theme',
-		'/domains',
-	];
+	const allowedPaths = [ '/browsehappy', '/themes', '/theme' ];
```

Comment:
`/themes/:site` will redirect to `/log-in`. Is this something to consider? It seems like we don't care about the rest of the path, as long as it starts with one of our allowed list.

Reply:
> Is this something to consider

It will redirect after it redirects, so I think it's okay. We allow the rest of the path because we want to allow theme slugs